data-access-analysis-tools/index-nb-intro.rst

```diff
@@ -95,7 +106,7 @@ The best way to learn how to use a Jupyter Notebook is to open the first of the
 Jupyter notebooks provide "cells" within which you type either Python code or markdown language (for formatted text).
 Choose the cell to execute by clicking in it with your mouse (the cursor must be in the desired cell).
 Hold down the `shift` key and press either `enter` or `return` (depending on your keyboard type), and the contents of the cell will be executed.
```

Comment:
"... type), or click the 'Play' button (▶️) in the notebook toolbar, and the ..."